web/src/main/java/org/akaza/openclinica/control/admin/CreateXformCRFVersionServlet.java

```diff
@@ -340,7 +343,7 @@ private XformContainer parseInstance(String xform, Errors errors, Html html, Str
 
             for (XformItem xformItem : xformItems) {
                 for (XformGroup xformGroup : xformGroups) {
-                    if (xformItem.getItemGroup().equals(xformGroup.getGroupName())) {
+                    if (xformItem.getItemGroup() != null && xformItem.getItemGroup().equals(xformGroup.getGroupName())) {
```

Comment:
The code update for this review is in the following pull request:  https://github.com/OpenClinica/OpenClinica/pull/721